src/frontend/src/actions/index.ts

```diff
@@ -355,8 +355,7 @@ const mapDispatchToProps = (dispatch: Function) => {
             } else {
               const path = window.location.pathname.substring(0,
                 window.location.pathname.lastIndexOf("/"));
-              asyncAction(Services.login(username, password, reset,
-                  PRODUCTION ? `https://${window.location.host}${path}/cgi-bin/login2.cgi` : undefined)).then((response) => {
```

Comment:
Mostly good, but I'd the old code here -- as it lets us test on seashell-dev.student.cs.uwaterloo.ca